veggieseasons/lib/main.dart

```diff
@@ -11,6 +11,7 @@ import 'package:veggieseasons/screens/home.dart';
 import 'package:veggieseasons/styles.dart';
 
 void main() {
+  WidgetsFlutterBinding.ensureInitialized();
```

Comment:
Remind me -- which platform/call requires this to be up front? Shared prefs plugin?

Reply:
The following line spits out a warning at run time because it is being run prior to `runApp()`.